requirements.txt

```diff
@@ -1,3 +1,4 @@
 coala>=0.10.0.dev20170213201648
 typing>=3.5.3.0
 coala-bears>=0.10.0.dev20170215041744
+python-language-server
```

Comment:
Do we need to add the version in the file?

Reply:
Yes, I think we should have the current version (0.18.0) specifically of pyls for now.

Reply:
SGTM